hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/domain/subscription/validator/InflightSizeValidatorTest.groovy

```diff
@@ -72,7 +72,8 @@ class InflightSizeValidatorTest extends Specification {
 
         then:
         def exception = thrown(ConstraintViolationException)
-        exception.message == "serialSubscriptionPolicy.inflightSize: must be greater than or equal to 1"
+        exception.message == "serialSubscriptionPolicy.inflightSize: must be greater than or equal to 1" ||
+                "serialSubscriptionPolicy.inflightSize: musi być równe lub większe od 1"
```

Comment:
what is this translation ?

Reply:
I'm also surprised by it but it's result of failing tests locally. As I checked the only reason of failing test was...That the violation message wasn't in polish. My entire system works in english so I was double surprised - on remote most likekly it woud pass as before since the message would be in english. Done it at least to check if the proper changes that was taken do not breaks anything while building

Reply:
hmm this is due to jakarta message translation, you can try to enforce the language by adding the jvm property https://stackoverflow.com/questions/8809098/how-do-i-set-the-default-locale-in-the-jvm while running tests

Reply:
anyway please leave only eng message :)

Reply:
Understood and on it